pages/common/git-sed.md

```diff
@@ -0,0 +1,21 @@
+# git sync
+
+> Replace patterns in git-controlled files using sed.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-sed>.
+
+- Replace the string 'this' with 'that':
```

Comment:
In all files in the repository? Or in the current repository?

Reply:
Perhaps this better
```suggestion
- Replace the specified text in the current repository:
```